src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -644,6 +655,17 @@ public AppPage clickAndConfirm(WebElement elementToClick) {
     }
     
     /**
+     * Clicks the element and clicks 'Yes' in the follow up dialog box.
+     * Fails if there is no dialog box.
+     * @return the resulting page.
+     */
+    public AppPage clickNoOnModal() {
```

Comment:
I may be wrong, but it looks like these two methods you added before are no longer used?